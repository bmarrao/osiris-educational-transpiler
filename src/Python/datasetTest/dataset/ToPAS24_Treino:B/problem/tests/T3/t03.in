10 10
3 5 2 7 8 0 3 1 4 4 5 8 4 6 2 9 9 2 13 3
24 b 11 a 2 p 3 m 8 r 6 t 7 K 12 R 20 A 10 j
