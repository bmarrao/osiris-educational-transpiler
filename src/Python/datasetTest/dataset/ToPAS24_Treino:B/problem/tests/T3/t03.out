5 5 5
7 7
0 0 0 0 0 0 0 0
1 1 1
4 4 4 4
8 8 8 8 8
6 6 6 6
9 9
2 2 2 2 2 2 2 2 2
3 3 3 3 3 3 3 3 3 3 3 3 3
b b b b b b b b b b b b b b b b b b b b b b b b
a a a a a a a a a a a
p p
m m m
r r r r r r r r
t t t t t t
K K K K K K K
R R R R R R R R R R R R
A A A A A A A A A A A A A A A A A A A A
j j j j j j j j j j
